code/datums/components/riding/riding_mob.dm

```diff
@@ -255,11 +261,11 @@
 
 /datum/component/riding/creature/human/get_offsets(pass_index)
 	var/mob/living/carbon/human/H = parent
-	if(H.buckle_lying)
-		return list(TEXT_NORTH = list(0, 6), TEXT_SOUTH = list(0, 6), TEXT_EAST = list(0, 6), TEXT_WEST = list(0, 6))
+	if(H.buckle_lying) //SKYRAT EDIT BEGIN - Oversized Overhaul
```

Comment:
Put the SKYRAT EDIT BEGIN comment on an empty line before your changes, that's better for Git blames.